CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Lotus changelog
 
+# 0.7.2 / 2020-09-23
+
+This optional release of Lotus introduces a major refactor around how a Lotus node interacts with code from the specs-actors repo. We now use interfaces to read the state of actors, which is required to be able to reason about different versions of actors code at the same time.
+
+Additionally, this release introduces various 
```

Comment:
...

Reply:
Maybe a "test carefully" warning?

Reply:
i obviously meant to leave that sentence trailing, it's